examples/vision/md/randaugment.md

```diff
@@ -0,0 +1,421 @@
+Test accuracy: 44.02%
+
+```
+</div>
+# 2. Train model with `simple_aug`
+
+
+```python
+simple_aug_model = get_training_model()
+simple_aug_model.load_weights("initial_weights.h5")
+simple_aug_model.compile(
+    loss="sparse_categorical_crossentropy", optimizer="adam", metrics=["accuracy"]
+)
+simple_aug_model.fit(train_ds_simple, validation_data=test_ds, epochs=EPOCHS)
+_, test_acc = simple_aug_model.evaluate(test_ds)
+print("Test accuracy: {:.2f}%".format(test_acc * 100))
+```
+
+<div class="k-default-codeblock">
+```
+391/391 [==============================] - 31s 69ms/step - loss: 1.7836 - accuracy: 0.3788 - val_loss: 1.3316 - val_accuracy: 0.5205
+79/79 [==============================] - 2s 22ms/step - loss: 1.3316 - accuracy: 0.5205
+Test accuracy: 52.05%
+
+```
+</div>
+---
+## Load the CIFAR-10-C dataset and evaluate performance
+
+
+```python
+# Load and prepare the CIFAR-10-C dataset
+# (If it's not already downloaded, it takes ~10 minutes of time to download)
+cifar_10_c = tfds.load("cifar10_corrupted/saturate_5", split="test", as_supervised=True)
+cifar_10_c = cifar_10_c.batch(BATCH_SIZE).map(
+    lambda x, y: (tf.image.resize(x, (IMAGE_SIZE, IMAGE_SIZE)), y),
+    num_parallel_calls=AUTO,
+)
+
+# Evaluate `rand_aug_model`
+_, test_acc = rand_aug_model.evaluate(cifar_10_c, verbose=0)
+print(
+    "Accuracy with RandAugment on CIFAR-10-C (saturate_5): {:.2f}%".format(
+        test_acc * 100
+    )
+)
+
+# Evaluate `simple_aug_model`
+_, test_acc = simple_aug_model.evaluate(cifar_10_c, verbose=0)
+print(
+    "Accuracy with simple_aug on CIFAR-10-C (saturate_5): {:.2f}%".format(
+        test_acc * 100
+    )
+)
+```
+
+<div class="k-default-codeblock">
+```
+Accuracy with RandAugment on CIFAR-10-C (saturate_5): 34.14%
+Accuracy with simple_aug on CIFAR-10-C (saturate_5): 46.90%
+
+```
+</div>
+For the purpose of this example, we trained the models for only a single epoch. In my
+experiments, I found that with RandAugment the model performs way better (76.64%) than
+the model trained with `simple_aug` (64.80%) on the CIFAR-10-C dataset. Additionally, I
+found that RandAugment helped stabilize the training. You can my experimentation notebook
+[here](https://nbviewer.jupyter.org/github/sayakpaul/Keras-Examples-RandAugment/blob/main/RandAugment.ipynb).
+
+As we can see from the notebook, at the expense of increased training time with RandAugment,
+we are able to carve out far better performance on the CIFAR-10-C dataset. You can
+experiment on the other corruption and perturbation settings that come with the
+run the same CIFAR-10-C dataset and see if RandAugment helps.
+
+You can also experiment with the different values of `n` and `m` in the `RandAugment`
+object. In the [original paper](https://arxiv.org/abs/1909.13719), the authors show
+the impact of the individual augmentation transforms for a particular task and a range of
+ablation studies. You are welcome to check them out.
```

Comment:
```suggestion
ablation studies.
```

("You are welcome to check them out." may not be necessary, since you already provided the link to that paper at least 2x) :)